config/initializers/devise.rb

```diff
@@ -250,7 +250,16 @@
   # Add a new OmniAuth provider. Check the wiki for more information on setting
   # up on your models and hooks.
   # config.omniauth :github, 'APP_ID', 'APP_SECRET', scope: 'user,public_repo'
-
+  config.omniauth :shibboleth,
+                  # TODO: mappings will need to change when authenticating against IST's SAML
+                  uid_field: 'eppn',
```

Comment:
The main difference with IST is that the uid_field in their response is 'uid' rather than 'eppn', and it comes in without '@domain.tld'. 

We use a lambda to dynamically choose which in current hydranorth (https://github.com/ualbertalib/HydraNorth/blob/master/config/initializers/devise.rb#L242) because the dev environment is doubling up between local dev (testshib/eppn) and a remote dev server that talks to IST's staging env (which passes UID).

We can/should make the remote dev server a separate environment, which means we really just need this block for dev and a second block for any environment for use with IST, where this is 'uid'. 

At the very least for this pull request I think we should chop the '@domain.tld' off of testshib's eppn response, so that during dev we don't assume it's a full address and then run into a surprise when we transition to IST's shib server and uids suddenly don't have that.